packages/clay-color-picker/src/Editor.tsx

```diff
@@ -153,8 +156,32 @@ export function Editor({
 		[b, 'b'],
 	];
 
+	const [alpha, setAlpha] = useState<number>(1);
+
+	React.useEffect(() => {
+		let currentAlpha = color.getAlpha();
+
+		if (currentAlpha !== 0 && currentAlpha !== 1) {
+			currentAlpha = +currentAlpha.toFixed(2);
```

Comment:
Since we can remove that here, you can still do that in the `onChange` of the `<Alpha />` component you are defining in the Editor.

Reply:
Removed this `useEffect` and moved this logic to `<Alpha />` component